### PR TITLE
Bug fix on bug fix ... throttling cast to int at the wrong place

### PR DESCRIPTION
**Pull request by PeterDM65**
https://github.com/mtaheij/PoGo-UWP/pull/185

### CODE REVIEW

**Comment by Furtif** [comment]
see ok 👍

File: POGOLib.Core/Net/RpcClient.cs [hunk 1]
```diff
@@ -515,7 +515,7 @@ private Task<ByteString> SendRemoteProcedureCall(RequestEnvelope requestEnvelope
                     while (_rpcQueue.TryDequeue(out processRequestEnvelope))
                     {
 //                        var diff = Math.Max(0, DateTime.Now.Millisecond - LastRpcRequest.Millisecond);
-                        var diff = Math.Min((int)(DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
+                        var diff = (int)Math.Min((DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
```
Comment (Furtif) [inline POGOLib.Core/Net/RpcClient.cs:518]: maybe 
```
int diff = 0; 

diff = (int)Math.Min((DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
if (diff < 0) diff = 0;
``` 
idee only

Comment (PeterDM65) [reply POGOLib.Core/Net/RpcClient.cs:518]: Under normal conditions LastRpcRequest always comes before UtcNow => So the difference should always be greater than zero. 
The problem here was that initial LastRpcRequest has a value of '01/01/0001' so the difference between the current date and LastRpcRequest was a big double value, and casting that number to a int resulted in a negative value.

**Review by Furtif — COMMENTED** [review]
Ok

**Pull request merged**